src/webui/api/apierror.h

```diff
@@ -36,7 +36,8 @@ enum class APIErrorType
     BadData,
     NotFound,
     AccessDenied,
-    Conflict
+    Conflict,
+    Unauthorized
```

Comment:
Ideally this list should be sorted too but it is already messed up before this PR, so I won't insist here.